vMenu/MainMenu.cs

```diff
@@ -118,6 +128,26 @@ public MainMenu()
             }
             #endregion
 
+            #region loading language
+            LanguageData = ConfigManager.GetLanguages();
+
+            string[] LanguageType = { "american", "french", "german", "italian", "spanish", "portuguese", "polish", "russian", "korean", "chinesetraditional", "japanese", "mexican", "chinesesimplified" };
+            string LanguageName = "";
+            
+            try {
+                LanguageName = LanguageType[API.GetCurrentLanguage()];
+                if (LanguageData.TryGetValue(LanguageName, out CurrentLanguage))
+                {
+                    Debug.WriteLine($"[vMenu] Successful load the language: {LanguageName}.");
```

Comment:
Please use the Log method from inside CommonFunctions to log non-errors on the client side. This will make it only log when client debugging mode is enabled.